ex7/s_tokida/ex7.py

```diff
@@ -0,0 +1,252 @@
+import kmeans
+import argparse
+import time
+import numpy as np
+import pandas as pd
+from scipy.stats import multivariate_normal
+from matplotlib import pyplot as plt
```

Comment:
PEP8に準拠したimport文をコーディングすると良いです．
isortという便利なPythonライブラリがあるので，よかったら使ってみてください．
自動でimport文をコーディング規約に沿って並び替えてくれます．
```suggestion
import argparse
import time

import kmeans
import numpy as np
import pandas as pd
from matplotlib import pyplot as plt
from scipy.stats import multivariate_normal
```